cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterPlanExecutor.java

```diff
@@ -107,9 +108,6 @@ public QueryDataSet processQuery(PhysicalPlan queryPlan, QueryContext context)
     ExecutorService pool = new ScheduledThreadPoolExecutor(THREAD_POOL_SIZE);
 
     for (Node node : metaGroupMember.getAllNodes()) {
-      if (node.equals(metaGroupMember.getThisNode())) {
-        continue;
-      }
```

Comment:
Why not skip the local node?

Reply:
I thought maybe I can use `client == null` to check if it is the local node. Since it makes no differentce, I use the `if (node.equals(metaGroupMember.getThisNode()))` as before.